app/controllers/sessions_controller.rb

```diff
@@ -1,4 +1,5 @@
 class SessionsController < ApplicationController
   def new
+
```

Comment:
Is this controller being used?  If not, we might as well delete it.

Reply:
Yeah - We can probably get rid of that controller